cip/cip67/src/main/java/com/bloxbean/cardano/client/cip/cip67/CIP67.java

```diff
@@ -0,0 +1,82 @@
+package com.bloxbean.cardano.client.cip.cip67;
+
+import com.bloxbean.cardano.client.crypto.CRC8;
+import com.bloxbean.cardano.client.util.IntUtil;
+
+import java.nio.ByteBuffer;
+
+public class CIP67 {
```

Comment:
Thanks @Kammerlo 

Should we rename CIP67 to CIP67AssetName or CIP67AssetNameUtil to give some context ?